.github/workflows/main.yml

```diff
@@ -15,15 +15,15 @@ jobs:
 
     strategy:
       matrix:
-        otp_version: ['23.1', '22.3.4.2', '21.3.8.16', '20.3.8.21']
+        otp_version: ['23', '22', '21']
```

Comment:
can we add back R20 now?

Reply:
If `hackney`'s no longer forcing a min. of 21, sure.

Reply:
Why add 20?